src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1833,7 +1833,7 @@ limitations under the License.
                 <requireCommand>GetSceneMembership</requireCommand>
                 <requireCommand>GetSceneMembershipResponse</requireCommand>
             </include>
-            <include cluster="Groups" client="false" server="true" clientLocked="true" serverLocked="true">
+            <include cluster="Groups" client="false" server="true" clientLocked="true" serverLocked="false">
```

Comment:
Groups Server, Conformance -> Awake. What does it mean?

Reply:
Optional but recommend ?

Reply:
Awake is optional.